controllers/openshift_controller.go

```diff
@@ -90,6 +91,8 @@ const (
 	peerpodsRuntimeClassName        = "kata-remote"
 	peerpodsRuntimeClassCpuOverhead = "0.25"
 	peerpodsRuntimeClassMemOverhead = "120Mi"
+	// cloud-api-adaptor (CAA) daemonset name
+	caaDsName = "osc-caa-daemon"
```

Comment:
nit: It might be a good idea to name it as daemonset or ds to be clear when looking at the pod o/p in CLI
`s/osc-caa-daemon/osc-caa-ds`
This is not a blocker